### PR TITLE
Editing the prodcli installation method

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

Adding new attributes, adding notes for OpenShift installation methods, fixing the used links

### CODE REVIEW

File: src/main/pages/che-7/overview/assembly_installing-che-on-openshift-3-using-the-operator.adoc [hunk 1]
```diff
@@ -29,6 +29,8 @@ Operators are a method of packaging, deploying, and managing a Kubernetes applic
 
 This chapter describes how to install {prod-short} on OpenShift 3, with the CLI management tool, using the Operator method.
 
+NOTE: This approach is only supported for use with {ocp} and {osd} version 3.11, but may also work for newer versions of {ocp} and {osd}.
```
Comment (rkratky) [inline src/main/pages/che-7/overview/assembly_installing-che-on-openshift-3-using-the-operator.adoc]: Don't use 'may'. It either works, or not. I thought we were treating it as a back up method.

**Review by rkratky — APPROVED** [review]

File: src/main/pages/che-7/overview/assembly_installing-che-on-openshift-4-from-operatorhub.adoc [hunk 2]
```diff
@@ -29,6 +29,14 @@ Operators are a method of packaging, deploying, and managing a Kubernetes applic
 
 On OpenShift, {prod} can be installed using the OperatorHub Catalog present in the OpenShift web console.
 
+[NOTE]
+====
+It is possible to use the `{prod-cli}` utility script for deploying {prod-short} on {ocp} and {osd} versions {oc4-ver}. This method is considered unofficial and serves as a back-up installation method for situations where the installation method using OperatorHub is not available.
```
Comment (yhontyk) [inline src/main/pages/che-7/overview/assembly_installing-che-on-openshift-4-from-operatorhub.adoc]:
```suggestion
It is possible to use the `{prod-cli}` utility script for deploying {prod-short} on {ocp} and {osd} versions {oc4-ver}. This method is considered unofficial and serves as a backup installation method for situations where the installation method using OperatorHub is not available.
```

Comment (yhontyk) [reply src/main/pages/che-7/overview/assembly_installing-che-on-openshift-4-from-operatorhub.adoc]: backup - noun, adjective
back up - verb

File: src/main/pages/che-7/overview/assembly_installing-che-on-openshift-4-from-operatorhub.adoc [hunk 3]
```diff
@@ -29,6 +29,14 @@ Operators are a method of packaging, deploying, and managing a Kubernetes applic
...
+
+For information how to use the `{prod-cli}` utility script for deploying {prod-short} on OpenShift, see the
```
Comment (yhontyk) [inline src/main/pages/che-7/overview/assembly_installing-che-on-openshift-4-from-operatorhub.adoc]: "_about_ how to use" ?

**Review by yhontyk — APPROVED** [review]
Just a few tiny comments. Otherwise, LGTM

**Pull request merged**